### DISCUSSION TITLE
HOW  to install packages when use pyodide  through whl

### DISCUSSION THREAD

**Discussion by wanghaisheng** [pyscript/pyscript#1769] [Q&A]
![image](https://github.com/pyscript/pyscript/assets/2363295/4f4c7d34-b18d-496a-a11e-6ecd3d336f59)
toml file
packages = ["playwright" ]
you can find error press F12 here 
https://wanghaisheng.github.io/run-python-wasm/pyscript-demo-pl/index.html

also tried in pyodide

![screenshot-20230929-015335](https://github.com/pyscript/pyscript/assets/2363295/901979e9-6136-4498-92d1-774cf85ffd89)

**Comment by JeffersGlass** [comment]
The Pyodide Terminal emulator is provided by [Pyodide](https://github.com/pyodide/pyodide), and is not affiliated with the PyScript project. If you're having an issue with the emulator, you'd be better off asking your question there.

Though I see you've also linked to a project using PyScript, so  guess I'm not sure I understand what your question is.
